app/assets/javascripts/client/directives/learning_questions/learningQuestionModal.js

```diff
@@ -0,0 +1,24 @@
+(function() {
+  'use strict';
+  angular.module('PDRClient')
+      .directive('learningQuestionModal', learningQuestionModal);
+
+  function learningQuestionModal() {
+    return {
+      restrict: 'E',
+      scope: {
+        reminder: '@'
+      },
+      templateUrl: 'client/views/modals/learning_questions/add_learning_question.html',
+      link: function(scope) {
+        // The modal exists on the parent scope, so we have to reach up to it  to bind any closing actions on the modal.
+        var parentScope = scope.$parent;
```

Comment:
Under what circumstances would this break if the parent doesn't contain the modal? I'm afraid this explicitness would limit the way we use this specific directive, it's making a lot of assumptions about the parent scope, I'm not sure if this is natural to Angular.js but in other lands isn't this called feature envy?